code/modules/clothing/head/jobs.dm

```diff
@@ -146,32 +158,56 @@
 	strip_delay = 60
 	dog_fashion = /datum/dog_fashion/head/beret/sec
 
+	sprite_sheets = list(
+		"Vox" = 'icons/mob/clothing/species/vox/head.dmi'
+		)
```

Comment:
This is inherited from the parent. Please remove the re-declaration for all the subtypes of 
`/obj/item/clothing/head/beret` beret as well.
```suggestion
```

Reply:
Done, sorry for being dumb